docs/installation/docker-compose.md

```diff
@@ -64,51 +59,61 @@ There is also a legacy project called [Docker Toolbox](https://docs.docker.com/t
+
+!!! Warning "Demonstration Purposes Only!"
+    As the name implies, `make demo` is for demonstration purposes only. If you want to add new modules and enable them, you will need to do so using `docker-compose exec drupal with-contenv bash -lc 'YOUR COMMAND'`. Both `drush` and `composer` can be used in this manner. 
+
+### Local
+
+`make local` creates an Islandora instance with a Drupal site from your computer.  This lets you use tools from your machine, such as an IDE or `composer` on your Drupal site, and have the results reflected automatically in your repository.
+
+If you provide a Drupal site in the `codebase` folder before running the command, it will be used to build your Drupal site. You can use either an entire Drupal site using git, or at a minimum, just a `composer.json` file. If your Drupal site has had its configuration exported using `drush`, you'll need to configure `isle-dc` to use it by editing your `.env` file and setting
+
+```
+INSTALL_EXISTING_CONFIG=true
+DRUPAL_INSTALL_PROFILE=minimal
+```
+
+If you already have a Drupal site but don't know how to export it,
+log into your server, navigate to the Drupal root, and run the following commands:
 
-Results of `make demo`:
+- `drush config:export`
+- `git init`
+- `git add -A .`
+- `git commit -m "First export of site"`
+
+Then you can `git push` your site to Github and `git clone` it down whenever you want. For
+example, from within your `isle-dc` directory:
+    
 ```
-isle-dc$ make demo
-  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
-                                 Dload  Upload   Total   Spent    Left  Speed
-100  3562  100  3562    0     0  12779      0 --:--:-- --:--:-- --:--:-- 12812
-  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
-                                 Dload  Upload   Total   Spent    Left  Speed
-100  1708  100  1708    0     0   6809      0 --:--:-- --:--:-- --:--:--  6832
-docker-compose pull
-Pulling activemq   ... done
-Pulling alpaca     ... done
-Pulling blazegraph ... done
-Pulling cantaloupe ... done
-Pulling fcrepo     ... done
-Pulling fits       ... done
-Pulling crayfits   ... done
-Pulling gemini     ... done
-Pulling homarus    ... done
-Pulling houdini    ... done
-Pulling hypercube  ... done
-Pulling mariadb    ... done
-Pulling matomo     ... done
-Pulling milliner   ... done
-Pulling recast     ... done
-Pulling solr       ... done
-Pulling drupal     ... done
-Pulling traefik    ... done
-Pulling watchtower ... done
+git clone https://github.com/your_org/your_repo.git codebase
+make local
 ```
 
+If you do not provide a `codebase` folder before running `make local`, you will be given a vanilla Drupal 9 installation with the `islandora` module enabled and only the iminimal configuration required to run.  It's not for the faint of heart, but if you want to avoid using `islandora_defaults`, you can build up your entire repository this way.
```

Comment:
typo, minimal.

Also, it appears the expected workflow for anyone doing "real site building" (not just kicking the tyres) starting out is to "already have a site" and I'm confused at the catch-22 here.

Reply:
What would the recommendation be for a group just starting with Islandora for a production environment? As @rosiel eluded to this is confusing. 

For example, we are looking to start using Islandora, but want a starting point that we can customize but not be a full blown demo site with non affiliated data. Thanks!